src/data/ExcelPopulateRepository.ts

```diff
@@ -38,7 +38,7 @@ export class ExcelPopulateRepository extends ExcelRepository {
             const id = idFromDefinedName || idFromCell;
 
             if (!id || typeof id !== "string") throw new Error("Invalid id");
-            cleanId = id.replace(/^.*?:/, "").trim();
+            cleanId = id.replace(/^.*?___/, "").trim();
```

Comment:
That's something we had already detected, unfortunately the PR where this is fixed was not yer mergeable (/feature/default-custom-templates-865bud46a). We will try to merge it as soon as possible, we've been working on that this week. Note that we using only underscore, but more importantly, sheetBuilder also needs to be updated. See:

https://github.com/EyeSeeTea/Bulk-Load/blob/feature/default-custom-templates-865bud46a/src/data/ExcelPopulateRepository.ts#L22

https://github.com/EyeSeeTea/Bulk-Load/blob/feature/default-custom-templates-865bud46a/src/webapp/logic/sheetBuilder.ts#L141

If you need this ASAP for your project, cherry pick this changes from the PR. Otherwise, you can wait for it to merged to development.